widgets/executions/src/backend.js

```diff
@@ -107,15 +108,25 @@ module.exports = r => {
 
         const runGraphCreation = () => {
             const tasksGraphParams = { ...req.query };
+            const headers = _.pick(req.headers, ['authentication-token', 'tenant']);
 
-            let { headers } = req;
             const operationsList = [];
             helper.Manager.doGet(tasksGraphsFetchUrl, tasksGraphParams, headers)
                 .then(data => {
-                    headers = _.omit(headers, 'accept-encoding'); // Required otherwise the data is returned in bytes and screwed up
-                    const operationsPromises = _.map(data.items, graph =>
+                    const { items } = data;
+
+                    if (_.isEmpty(items)) {
+                        const { execution_id: id } = tasksGraphParams;
```

Comment:
Nothing important but I find this notation easier to read, applies to other places as well.
```suggestion
                        const id = tasksGraphParams.execution_id;
```

Reply:
Done.